src/app/exercise/editors/editors.component.ts

```diff
@@ -20,6 +20,29 @@ export class EditorsComponent {
     this.debug = true;
   }
 
+  ngOnInit() {
+    this.files.map( file => {
+      file.opened = !file.readonly;
+    });
+  }
+
+  closeTab(file) {
+    file.opened = false;
```

Comment:
Probably naming is confusing. Will it be better if I rename `isOpenFile` to `isActiveFile`. So it will be clear that `isActive` checks if the file is being viewed? 
Opened are the files that user selected and are displayed in tabs. Active is currently viewed.